config/config.yaml

```diff
@@ -0,0 +1,2 @@
+ENV_PRODUCT: rke2
```

Comment:
we should add on the README. , the informations to running this like....rke2 should not be hardened and etc ...